### PR TITLE
gh-126882: Fix indentation in code sample block

### PR DESCRIPTION
**Pull request by javert42**
Both code blocks are text from a Python REPL session. **pycon** is an alias for the Python REPL language used for syntax highlighting. The problem is that the lines all begin with unnecessary spaces, while **pycon** expects each line of input to begin with '^>>>' or '^...' (see link below).

Changing the language type to **python** would also work, but **pycon** seems to be appropriate for this case (after getting rid of the leading spaces).

This change keeps the language type (**pycon**) and removes the leading spaces.

Notes:
I believe this is what is being used for syntax highlighting in both GitHub and VS Code:
https://github.com/highlightjs/highlight.js/blob/c49081a84fc5de213383287b68ce2ecc238e537d/src/languages/python-repl.js

* Issue: gh-126882

### CODE REVIEW

**Comment by tomasr8** [comment]
Thanks! There's a couple more code blocks in the file that should be fixed as well

**Comment by javert42** [comment]
I changed the language type from **pycon** to **python** and got rid of the leading spaces in all of the code blocks in this document.

One of the code blocks that I missed in my first commit has line-comments inserted between lines of the REPL session. Using **pycon** wouldn't highlight these comments because their lines don't start with **>>>**. Adding the **>>>** in front of every comment would correct the syntax, and enable comment highlighting, but it gets a bit messy. Just changing the language type to **python** provides the expected syntax-highlighting, and allows the author a little license to stray from strict representation of an actual REPL session.

File: InternalDocs/garbage_collector.md [hunk 1]
```diff
@@ -16,27 +16,27 @@ count field can be examined using the `sys.getrefcount()` function (notice that
...
-    >>> x = object()
-    >>> sys.getrefcount(x)
-    2
-    >>> y = x
-    >>> sys.getrefcount(x)
-    3
-    >>> del y
-    >>> sys.getrefcount(x)
-    2
+```python
```
Comment (skirpichev) [inline InternalDocs/garbage_collector.md]: This is wrong.  It's actually an example of console session.  Please revert.  (Ditto other cases.)

**Review by skirpichev — CHANGES_REQUESTED** [review]

File: InternalDocs/garbage_collector.md (hunk 1 above)
Comment (javert42) [reply InternalDocs/garbage_collector.md]: Are you referring to the indentation change too, or just the **pycon** -> **python** change?

What's your opinion on the issue with line-comment highlighting that I mentioned in my previous comment? Non-issue?

Comment (skirpichev) [reply InternalDocs/garbage_collector.md]: > Are you referring to the indentation change too

No, that part seems fine.

> What's your opinion on the issue with line-comment highlighting that I mentioned in my previous comment?

Seems to be a minor issue.

You might either prepend missing ">>> " or reformat some comments to be *after* code.  See e.g. the tutorial: https://docs.python.org/3/tutorial/introduction.html#lists

File: InternalDocs/garbage_collector.md [hunk 2]
```diff
@@ -448,31 +448,23 @@ The content of these generations can be examined using the
...
+```pycon
 >>> import gc
 >>> class MyObj:
 ...     pass
 ...
-
-# Move everything to the old generation so it's easier to inspect
-# the young generation.
-
+>>> # Move everything to the old generation so it's easier to inspect the young generation.
```
Comment (skirpichev) [inline InternalDocs/garbage_collector.md]: This line is too long.  Ditto similar case below.  Just don't reformat comments.

**Review by skirpichev — APPROVED** [review]

**Comment by skirpichev** [comment]
@javert42, [please avoid using "update branch" button](https://devguide.python.org/getting-started/pull-request-lifecycle/#update-branch-button) without a good reason.  Obviously, it can be used to trigger CI run, or when you resolve merge conflicts.

**Comment by javert42** [comment]
> @javert42, [please avoid using "update branch" button](https://devguide.python.org/getting-started/pull-request-lifecycle/#update-branch-button) without a good reason. Obviously, it can be used to trigger CI run, or when you resolve merge conflicts.

Sorry about that. I had read that documentation, but I thought the update was necessary to unblock the merge. It sounds like I'm not involved in anything beyond this point (assuming no addition change requests or merge conflicts).

Thanks for all your feedback.

**Review by tomasr8 — APPROVED** [review]
LGTM! Thanks @javert42!

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**